objcopy.bzl

```diff
@@ -0,0 +1,82 @@
+# SPDX-FileCopyrightText: 2023 Rivos Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+load(
+    "@bazel_tools//tools/cpp:toolchain_utils.bzl",
+    "find_cpp_toolchain",
+    "use_cpp_toolchain",
+)
+
+def _objcopy_elf_to_bin_impl(ctx):
```

Comment:
more accurately, this is `objcopy_to_elf` or `objcopy_to_object`.

Reply:
fixed